operator_service/routes.py

```diff
@@ -218,21 +218,19 @@ def stop_compute_job():
 
         if not owner or len(owner) < 2:
             owner = None
-
+        #msg, status = process_signature_validation(data.get('providerSignature'), agreement_id)
+        #if msg:
+        #    return jsonify(error=f'`providerSignature` of agreementId is required.'), status
         if owner is None and agreement_id is None and job_id is None:
             msg = f'You have to specify one of agreementId, jobId or owner'
             logging.error(msg)
             return jsonify(error=msg), 400
-
-        msg, status = process_signature_validation(data.get('providerSignature'), agreement_id)
-        if msg:
-            return jsonify(error=f'`providerSignature` of agreementId is required.'), status
-
         jobs_list = get_sql_jobs(agreement_id, job_id, owner)
-        for ajob in jobs_list:
-            name = ajob
-            logging.info(f'Stopping job : {name}')
-            stop_sql_job(name)
+        if jobs_list is not None:
```

Comment:
It looks cleaner if `get_sql_jobs` always returned a list, then this check is not necessary.